### ISSUE TITLE
[Bug] No 404 page due to empty layouts/404.html file

### ISSUE THREAD

**Issue by rajeshduggal** [mattstratton/castanet#150]
### Expected behavior
Website should have a 404 not found page.
### Actual behavior
No 404 file so generic unthemed webserver 404 error page displayed.,
### Reproduction Steps
Install theme into website, run hugo. Notice no public/404.html present.
### Operating System and Version
Windows 7 (Cygwin)
### Web Browser and Version
n/a
### Hugo Version
Hugo v0.36.1 windows/amd64

**Comment by rajeshduggal** [comment]
I see on your website https://www.arresteddevops.com/missing-page it shows a themed webpage (strangely it only says "Not Found" when in responsive small width. but in desktop width it doesn't)... but for our website it's a bug, because we don't get a themed not found page (We see a github pages 404)...  https://khabardaarpodcast.com/missing-page.

**Comment by mattstratton** [comment]
Arrested DevOps isn’t using castanet yet :)
		
		

		Get Outlook for iOS
	




On Tue, Mar 6, 2018 at 11:26 AM -0600, "Rajesh Duggal" <notifications@github.com> wrote:












I see on your website https://www.arresteddevops.com/missing-page it shows a themed webpage (strangely it only says "Not Found" when in responsive small width. but in desktop width it doesn't)... but for our website it's a bug, because we don't get a themed not found page (We see a github pages 404)...  https://khabardaarpodcast.com/missing-page.

**Comment by mattstratton** [comment]
I could be wrong but I also vaguely remember something weird with hugo in general with regard to 404 pages on github pages (I host all my hugo stuff on netlify) but either way, we need a themed 404 (the template now is blank).

**Comment by mattstratton** [comment]
@rajeshduggal - for my knowledge, are you pulling the theme in as a submodule off of `master`? Just want to make sure I don't break things out from underneath you while I'm making updates :)

**Comment by mattstratton** [comment]
Reference for Matt - https://gohugo.io/templates/404/#readout

**Comment by rajeshduggal** [comment]
@mattstratton Yes I am pulling in the theme as a submodule. Which means that my website's git repo is tied to a particular sha of the castanet theme.. i.e. I don't get the newer castanet commits unless I explicitly choose to have them in my website.  So I've been reviewing the castanet commits before tying my repo to the latest castanet sha. So all good, thanks for checking in with me !

**Comment by ctindel** [comment]
Was this closed? I saw someone submitted a PR for it

**Comment by mattstratton** [comment]
This will be resolved in v2; issue wil close when that is released
